package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -269,6 +275,7 @@ hostapd_bss_del_client(struct ubus_context *ctx, struct ubus_object *obj,
 	struct sta_info *sta;
 	bool deauth = false;
 	int reason;
+	int max_ban_attempts;
```

Comment:
Can't you just set it to 0 here so you don't need that else block bellow (there is a stray white space after that else anyway)?